api/src/opentrons/hardware_control/backends/ot3controller.py

```
@@ -593,42 +597,45 @@ async def home(
         ]
         async with self._monitor_overpressure(moving_pipettes):
             positions = await asyncio.gather(*coros)
+        # TODO(CM): default gear motor homing routine to have some acceleration
         if Axis.Q in checked_axes:
             await self.tip_action(
-                [Axis.Q],
-                self.axis_bounds[Axis.Q][1] - self.axis_bounds[Axis.Q][0],
-                self._configuration.motion_settings.max_speed_discontinuity.high_throughput[
+                distance=self.axis_bounds[Axis.Q][1] - self.axis_bounds[Axis.Q][0],
+                velocity=self._configuration.motion_settings.max_speed_discontinuity.high_throughput[
                     Axis.to_kind(Axis.Q)
                 ],
+                tip_action="home",
             )
         for position in positions:
             self._handle_motor_status_response(position)
         return axis_convert(self._position, 0.0)
 
     async def tip_action(
         self,
-        axes: Sequence[Axis],
-        distance: float,
-        speed: float,
+        moves: Optional[List[Move[Axis]]] = None,
```

What happens when this function is called without moves nor distance/velocity? 

Instead of making these three args optional, we should probably separate them into two functions, one requiring moves and the other both distance and velocity

---

We could also just raise an error.

---

That's fine by me

---

Let's just raise an error for now but fix it Soon